CHANGELOG.md

```diff
@@ -8,6 +8,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [2.0.0] - 2020-11-20
```

Comment:
It doesn't look like we had breaking changes in the external interface, so I'd probably use `1.10.0`, but I'm OK if you prefer 2.0.0.

(the removal of `PrometheusRules` from per-cluster namespace is really just moving them, and it's not really an external interface, so seems more like refactoring)

Reply:
yeah you're right. I didn't think too much. I saw, removed and I bumped major version

Reply:
closing and going with 1.10.0 https://github.com/giantswarm/prometheus-meta-operator/pull/216